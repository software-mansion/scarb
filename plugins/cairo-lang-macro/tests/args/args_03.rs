use cairo_lang_macro::{inline_macro, ProcMacroResult, TokenStream, MACRO_DEFINITIONS_SLICE};

#[inline_macro(parent = "parent")]
fn t1(_a: TokenStream) -> ProcMacroResult {
    ProcMacroResult::new(TokenStream::empty())
}

fn main() {}
