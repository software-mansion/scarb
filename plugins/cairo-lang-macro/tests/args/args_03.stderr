error: inline macro cannot use `parent` argument
 --> tests/args/args_03.rs:3:25
  |
3 | #[inline_macro(parent = "parent")]
  |                         ^^^^^^^^

warning: unused imports: `MACRO_DEFINITIONS_SLICE`, `ProcMacroResult`, and `TokenStream`
 --> tests/args/args_03.rs:1:38
  |
1 | use cairo_lang_macro::{inline_macro, ProcMacroResult, TokenStream, MACRO_DEFINITIONS_SLICE};
  |                                      ^^^^^^^^^^^^^^^  ^^^^^^^^^^^  ^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: `#[warn(unused_imports)]` on by default
